electron note (#5168)
braze-inc/braze-docs: _docs/_developer_guide/platform_integration_guides/web/initial_sdk_setup.md

},
```

### Electron support {#electron}

> Electron does not officially support web push notifications (see: this [Github issue](https://github.com/electron/electron/issues/6697)). There are other [open source workarounds](https://github.com/MatthieuLemoine/electron-push-receiver) that use Android FCM that have not been tested by Braze that you can explore.

Reviewer: I don't that `Android FCM` is the correct terminology

Author: ```suggestion
Electron does not officially support web push notifications (see: this [Github issue](https://github.com/electron/electron/issues/6697)). There are other [open source workarounds](https://github.com/MatthieuLemoine/electron-push-receiver) you may try that have not been tested by Braze.
```
